src/main/java/com/createment/springone/ApiController.java

```diff
@@ -10,11 +10,11 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-
+//adding comment
 @RestController
 @RequestMapping("/api")
 public class ApiController {
-    final String DB_HOST = "jdbc:mysql://localhost:3306/bookstore";
+    final String DB_HOST = "jdbc:mysql://192.168.1.191:3306/bookstore";
```

Comment:
No, it needs to be local

Reply:
no! my docker is running on the laptop. im on my main PC. i NEED to use the internal ip to access the laptop